.github/workflows/wide_integer.yml

```diff
@@ -1,8 +1,11 @@
 name: wide_integer
-on:
+on: 
   push:
     branches:
       - '**'
+  pull_request:
```

Comment:
@ckormanyos I'm not sure this makes a difference: a PR never happens without a push.